src/vscode-bicep/src/deploy/azureClients.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { ResourceManagementClient } from "@azure/arm-resources";
+import { SubscriptionClient } from "@azure/arm-resources-subscriptions";
+import {
+  AzExtClientContext,
+  createAzureClient,
+  createAzureSubscriptionClient,
+} from "@microsoft/vscode-azext-azureutils";
+
+export async function createResourceManagementClient(
+  context: AzExtClientContext
+): Promise<ResourceManagementClient> {
+  return createAzureClient(
+    context,
+    (await import("@azure/arm-resources")).ResourceManagementClient
```

Comment:
Can't we just return `ResourceManagementClient` here since it's already imported at the top?

Reply:
Added comment here: https://github.com/Azure/bicep/commit/0f57eb1d57af701b59203d6224aa029f96bd2f0d. This s used to lazy-load @azure packages to improve startup performance.